app/models/hunter.rb

```diff
@@ -150,4 +151,8 @@ def unstable?
   def to_s
     name
   end
+
+  def add_defaults
+    self.moves = Moves::Basic.where(name: Moves::Basic.default_moves) if moves.empty?
```

Comment:
Why do we need the `where(name: Moves::Basic.default_moves)`? We should be adding all Moves with a type of `Moves::Basic`.

Reply:
Thank you for the feedback, the PR mentioned 9 moves but it seems there was more basic moves in the seed file which I was I took that approach

Reply:
Thank you for clarifying, and sorry for the confusion. Those extra moves are all Use Magic variants from the Tome of Mysteries, an extension, so I forgot about them. We should include those as well though. I think it'll be cleaner than trying to keep a running list.
https://github.com/ChaelCodes/HuntersKeepers/blob/1195c093618e7c3c701545d774067d1e6db62399/db/seeds/move.seeds.rb#L278-L293